discord_bots/utils.py

```diff
@@ -825,3 +829,174 @@ def get_team_name_diff(
+                                )
+
+        for player in team1_players:
+            if player.move_enabled and ds_voice_channel:
+                member: Member | None = guild.get_member(player.id)
+                if member:
+                    member_voice: VoiceState | None = member.voice
+                    if member_voice:
+                        if member_voice.channel:
+                            try:
+                                coroutines.append(
+                                    member.move_to(
+                                        ds_voice_channel,
+                                        reason=f"Game {game_id} started",
+                                    )
+                                )
+                            except Exception:
+                                _log.exception(
+                                    f"Caught exception moving player to voice channel"
+                                )
+    # use gather to run the moves concurrently in the event loop
+    # note: a member has to be in a voice channel already for them to be moved, else it throws an exception
+    results = await asyncio.gather(*coroutines, return_exceptions=True)
+    for result in results:
+        # results should be empty unless an exception occured when moving a player
+        if isinstance(result, BaseException):
+            _log.exception("Ignored exception when moving a gameplayer:")
+
+async def move_game_players_lobby(
+    game_id: str, guild: Guild
+):
+    session: sqlalchemy.orm.Session
+    with Session() as session:
+        in_progress_game: InProgressGame | None = (
+            session.query(InProgressGame)
+            .filter(InProgressGame.id == game_id)
+            .first()
+        )
+        if not in_progress_game:
+            return
+        
+        queue: Queue | None = (
+            session.query(Queue)
+            .filter(Queue.id == in_progress_game.queue_id)
+            .first()
+        )
+        if not queue or not queue.move_enabled:
+            return
+        
+        voice_lobby: discord.abc.GuildChannel | None = guild.get_channel(
+                config.VOICE_MOVE_LOBBY
+        )
+        if not isinstance(voice_lobby, VoiceChannel) or not voice_lobby:
+            _log.exception("VOICE_MOVE_LOBBY not found")
+            return
+
+        ipg_channels: list[InProgressGameChannel] | None = (
+                session.query(InProgressGameChannel)
+                .filter(InProgressGameChannel.in_progress_game_id == in_progress_game.id)
+                .all()
+            )
+        
+        coroutines = []
+        for ipg_channel in ipg_channels or []:
+            discord_channel: discord.abc.GuildChannel | None = guild.get_channel(
+                ipg_channel.channel_id
+            )
+            if isinstance(discord_channel, VoiceChannel):
+                members: list[Member] = discord_channel.members
+                if len(members) > 0:
```

Comment:
nit: check non-emptiness with `if members` instead. I don't think you need this check at all, since the for each loop does that for you. The only consideration where would be if the `members` list can be null, which I'm not sure it can be